resp/responses/sensors/lennartz.yaml

```diff
@@ -88,5 +99,16 @@ filter:
     delay: 0
     inputunits: m/s
     outputunits: V
+  LE-3D-BH-MkIII:
+  - type: paz
+    lookup: LE-3Dlite
+    frequency: 15
```

Comment:
Is frequency 15Hz or 5Hz?

Reply:
@CBurton90  will need to check that (since it s derived from the default delat information for LE3D_lite_ as opposed to BH